### DISCUSSION TITLE
I am getting a Forbidden Error

### DISCUSSION THREAD

**Discussion by Marc3y** [spotify-web-api-java/spotify-web-api-java#314] [Q&A]
If I am trying to make any request I get an Forbidden Error.
The access token is right (saved in a database) and it should have all scopes.

How can I fix this?

**Comment by dargmuesli** [comment]
This repository is not the Spotify Web API itself so I cannot know what's happening. This repository provides only a possible way to access the API. It seems like you are not allowed to access resources with the means provided. If you expect help, you would need to provide more information or a crystal ball 🔮😉

**Reply by Marc3y** [reply]
So I am coding a Twitch Bot where you should see the song when you enter !song. I saved the names and access token refresh token and everything (from spotify) in a database and whenever you enter this command, the song name should come up. because I rewrote the code a bit it looks like this: https://pastebin.com/vkNUEAbf

But I get the error "Error: Forbidden"

**Reply by dargmuesli** [reply]
Are you using the `user-read-currently-playing` scope?

**Reply by Marc3y** [reply]
yes this is my link https://accounts.spotify.com/de/authorize?client_id=38bd7544c7d04364941d952423e3023f&response_type=code&redirect_uri=https%3A%2F%2Fmultipixels.de%2Fmarceybot%2Fredirect%2Fspotify%2F&show_dialog=true&scope=ugc-image-upload%20user-read-playback-state%20app-remote-control%20user-modify-playback-state%20playlist-read-private%20user-follow-modify%20playlist-read-collaborative%20user-follow-read%20user-read-currently-playing%20user-read-playback-position%20user-library-modify%20playlist-modify-private%20playlist-modify-public%20user-read-email%20user-top-read%20streaming%20user-read-recently-played%20user-read-private%20user-library-read

**Reply by dargmuesli** [reply]
I can only say that something in this workflow seems to contain a hiccup. I don't see an obvious mistake immediately, but what the Spotify api says is quite right in basically all cases.

**Reply by Marc3y** [reply]
this is a method that i am using. https://pastebin.com/6Yt9mS53 is there a problem or something like that?
